rosie/src/main/java/com/karumi/rosie/domain/usecase/UseCaseHandler.java

```diff
@@ -29,9 +33,13 @@
   private final TaskScheduler taskScheduler;
   private final ErrorHandler errorHandler;
 
+  // This list is used to retain error adapters in a non-weak reference associated with this use case handler
+  private List<OnErrorCallbackToErrorHandlerAdapter> errorAdapters;
```

Comment:
keep the comment if you like it. We only add JavaDoc to the public methods and fields. We don't add javadoc to private code. 

If you want remove the comment the best option is move this code ``errorAdapters.add(errorAdapter);`` to a method ``keepStrongReference`` or ``keepReference`` but I think that it is unnecessary.